Python/cim-convert-tool/configEqCim17.py

```diff
@@ -0,0 +1,602 @@
+                'cim:Equipment.normallyInService': {'type': 'bool', 'list': False},
+                'cim:Switch.locked': {'type': 'bool', 'list': False},
+                'cim:Switch.normalOpen': {'type': 'bool', 'list': False},
+                'cim:Switch.ratedCurrent': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:CurrentFlow.value'},
+                'cim:Switch.retained': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+                'cim:Equipment.EquipmentContainer': {'rdf:resource': {'type': 'string'}, 'list': False},
+                'cim:ConductingEquipment.BaseVoltage': {'rdf:resource': {'type': 'string'}, 'list': False}
+            }
+        },
+        'cim:GeneratingUnit': {
+            'mainAttributes': {
+                'rdf:ID': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:IdentifiedObject.mRID': {'type': 'string', 'list': False},
+                # 'cim:IdentifiedObject.aliasName': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.description': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.name': {'type': 'string', 'list': False},
+                'cim:Equipment.aggregate': {'type': 'bool', 'list': False},
+                # 'cim:Equipment.networkAnalysisEnabled': {'type': 'bool', 'list': False},
+                'cim:Equipment.normallyInService': {'type': 'bool', 'list': False},
+                'cim:GeneratingUnit.highControlLimit': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.initialP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.lowControlLimit': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.maxEconomicP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.maxOperatingP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.minEconomicP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.minOperatingP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.nominalP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.ratedGrossMaxP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:GeneratingUnit.ratedNetMaxP': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'}
+            },
+            'attributes': {
+                'cim:Equipment.EquipmentContainer': {'rdf:resource': {'type': 'string'}, 'list': False}
+            }
+        },
+        'cim:ControlArea': {
```

Comment:
Defined in line 136. this should be removed

Reply:
Can you say more?

Reply:
Sorry. ControlArea seems to have been defined twice, here in line 262 and above in line 136. The definition in line 136 is the one that corresponds to what we have in our examples.

Reply:
Fixed